### DISCUSSION TITLE
v5.0.0-RC2

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#33] [General]
**重要功能性修复，请务必更新！**
- **重要修复**：**首页图片不取主题色的问题**
- **升级**：主题更新程序

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-RC2'>v5.0.0-RC2</a>.</em>

**Comment by liyunjian** [comment]
2020年左右，我购买了这个主题，在主题停止维护之后我也停止了WP的更新，为了避免出现bug。直到今天，我由于阿里云服务器到期且续费太贵不得不迁移到华为云。打算处理一下站点一些问题。
搜索了一下nirvana发现了这里还有人在维护。顺便更新了一下主题。流畅多了哈哈哈。

**Reply by michaelliunsky** [reply]
哈哈，遇到bug可以随时跟我说哦，等主题慢慢稳定了，发5.0正式版
